templates/openstack/features.html

```diff
@@ -2,79 +2,282 @@
 
 {% block title %}Features | OpenStack{% endblock %}
```

Comment:
Should the title not be: "Carrier-grade private cloud - Canonical’s Charmed OpenStack"?